### ISSUE TITLE
vax8600: says it can boot from tq0

### ISSUE THREAD

**Issue by b4** [simh/simh#334]
Using the vax8600 simulator,

HELP CPU indicates TQn, is bootable however:

sim> show ver
VAX 8600 simulator V4.0-0 Beta
       	Simulator Framework Capabilities:
       		64b data
       		64b addresses
       		Ethernet Packet transports:PCAP:TAP:NAT:UDP
       		Idle/Throttling support is available
       		Virtual Hard Disk (VHD) support
       		Asynchronous I/O support
       		FrontPanel API Version 2
       	Host Platform:
       		Compiler: GCC 4.2.1 Compatible Apple LLVM 7.3.0 (clang-703.0.31)
       		Simulator Compiled as C: Aug 29 2016 at 20:36:09
       		Memory Access: Little Endian
       		Memory Pointer Size: 64 bits
       		Large File (>2GB) support
       		SDL Video support: No Video Support
       		PCRE RegEx support for EXPECT commands
       		OS clock resolution: 1ms
       		Time taken by msleep(1): 2ms
       		OS: Darwin melanie-2358.local 15.6.0 Darwin Kernel Version 15.6.0: Thu Jun 23 18:25:34 PDT 2016; root:xnu-3248.60.10~1/RELEASE_X86_64 x86_64
        git commit id: 223f0807

**Comment by markpizz** [comment]
You've shown the version detail, but please provide details about exactly what doesn't work?

Config file?
Failing output produced?

**Comment by b4** [comment]
Let me rephrase - is TQ actually a bootable device on the 8600 simulator?

**Comment by markpizz** [comment]
Thanks for noticing this.
